src/backend/index.js

```diff
@@ -42,6 +42,7 @@ async function enqueueWikiFeed() {
 enqueueWikiFeed()
   .then(() => {
```

Comment:
Often this means you can just rely on the implicit return value you get with an arrow function.  I think this is better:

```
.then(() => feedWorker.start())
```